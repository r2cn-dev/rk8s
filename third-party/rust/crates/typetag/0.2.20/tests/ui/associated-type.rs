#[typetag::serde]
pub trait Trait {
    type Assoc;
}

fn main() {}
