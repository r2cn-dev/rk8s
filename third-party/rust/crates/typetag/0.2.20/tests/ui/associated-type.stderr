error: typetag trait with associated type is not supported yet
 --> tests/ui/associated-type.rs:3:5
  |
3 |     type Assoc;
  |     ^^^^^^^^^^^
